lib/Doctrine/ODM/MongoDB/Id/ObjectIdGenerator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ODM\MongoDB\Id;
+
+use Doctrine\ODM\MongoDB\DocumentManager;
+use MongoDB\BSON\ObjectId;
+
+/**
+ * AutoGenerator generates a native ObjectId
+ *
+ * @internal
+ */
```

Comment:
This comment isn't necessary, and the class is internal. Consistent with other classes.
```suggestion
/** @internal */
```

Reply:
Done